src/Form/Fields.php

```diff
@@ -36,19 +37,52 @@ public function __set(string $name, $field): void
 		parent::__set($field->name(), $field);
 	}
 
+	/**
+	 * Returns an array with the default value of each field
+	 */
+	public function defaults(): array
+	{
+		return $this->toArray(fn ($field) => $field->default());
+	}
+
+	/**
+	 * An array of all found in all fields errors
+	 */
+	public function errors(): array
+	{
+		$errors = [];
+
+		foreach ($this->data as $name => $field) {
+			if (empty($field->errors()) === false) {
```

Comment:
```suggestion
			if ($field->errors() !== []) {
```

Or using `count()`. But we should get away from using `empty()`.